src/System.Management.Automation/FormatAndOutput/common/TableWriter.cs

```diff
@@ -82,18 +82,6 @@ internal void Initialize(int leftMarginIndent, int screenColumns, Span<int> colu
             //Console.WriteLine("         1         2         3         4         5         6         7");
             //Console.WriteLine("01234567890123456789012345678901234567890123456789012345678901234567890123456789");
 
-            if (screenColumns == int.MaxValue)
-            {
-                try
-                {
-                    screenColumns = System.Console.WindowWidth;
-                }
-                catch
-                {
-                    screenColumns = 120;
-                }
-            }
```

Comment:
Why are we removing this part?

Reply:
It duplicates a code in `GetConsoleWindowWidth` so we should remove this (or add cache as in `GetConsoleWindowWidth`).

Reply:
OK, got it. #close